lib/rubocop/cop/base.rb

```diff
@@ -126,6 +127,36 @@ def self.support_multiple_source?
         false
       end
 
+      ## Gem requirements
+
+      # rubocop:disable Layout/ClassStructure
+      class << self
+        attr_reader :gem_requirements
```

Comment:
I'm not quite sure what the correct layout is.

I also see `class << self` is pretty rare in this codebase, but `singleton_class.attr_reader(:gem_requirements)` would be weird.